docs/FAQ.md

```diff
@@ -218,9 +218,17 @@ runners:
 ...
 ```
 
+## Monitoring
+This boilerplate provides two solutions for monitoring: 
+1. VictoriaMetrics based on [victoria-metrics-k8s-stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack)
+2. Prometheus based on [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)  
+
+VictoriaMetrics is installed by default. However, you can easily switch to Prometheus just **enabling** it and **disabling** VictoriaMetrics in `terraform/layer2-k8s/helm-releases.yaml`. You need to do it before first applying the layer2-k8s.  
```

Comment:
You need to do it before first applying the layer2-k8s. -> You need to do it before the first apply of the layer2-k8s.